go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -380,7 +380,17 @@ func (qre *QueryExecutor) execDDL(conn *StatefulConnection) (*sqltypes.Result, e
 		}
 	}()
 
-	result, err := qre.execStatefulConn(conn, qre.query, true)
+	sql := qre.query
```

Comment:
it would be good with a unit test for this logic

Reply:
Added in the last commit